source/reference/minio-mc-admin/mc-admin-user-svcacct-add.rst

```diff
@@ -84,18 +84,34 @@ Parameters
 .. mc-cmd:: --access-key
    :optional:
 
-   A string to use as the access key for this account. Omit to let MinIO autogenerate a random value.
+   A string to use as the access key for this account.
+   Omit to let MinIO autogenerate a random 20 character value.
 
    Access Key names *must* be unique across all users.
 
 .. mc-cmd:: --comment
    :optional:
 
-   .. versionadded:: RELEASE.2023-01-28T20-29-38Z
+   .. versionchanged:: RELEASE.2023-05-18T16-59-00Z
```

Comment:
Should there be both `versionadded` and `versionchanged` for this?

Reply:
I'd keep both.
Versionadded means it won't be present at all in releases older than that release.
Versionchanged means something about the functionality is different from that release on than what users might be accustomed to in an older version.

Reply:
I think I'd just keep the changed directive, but then say something like
```suggestion
   .. versionchanged:: RELEASE.2023-05-18T16-59-00Z
      Replaced by :mc-cmd:`~mc admin user svcacct add --description` and :mc-cmd:`~mc admin user svcacct add --name`.
      
      Originally added in version RELEASE.2023-01-28T20-29-38Z.

```

Reply:
Agree on dropping the added directive, to be clear.